changelog/7132.feature.rst

```diff
@@ -0,0 +1 @@
+Added two environment variables ``PYTEST_THEME`` and ``PYTEST_THEME_MODE`` to let the users customize the pygments theme used.
```

Comment:
I think it would be nice to add a link to the docs here (possibly just change the env vars to `:envvar:PYTEST_THEME`, I think).